app/models/provider.rb

```diff
@@ -276,13 +276,6 @@ def discard_courses
     courses.each(&:discard)
   end
 
-  VISA_SPONSORSHIP_REQUIRED_FROM = 2022
-  def visa_sponsorship_publishable?
-    return true if recruitment_cycle.year.to_i < VISA_SPONSORSHIP_REQUIRED_FROM
-
-    %i[can_sponsor_student_visa can_sponsor_skilled_worker_visa].none? { |attr| public_send(attr).nil? }
-  end
-
```

Comment:
I think if the refactored validation works this method is no longer required. Might be helpful to retain the constant (or a more generic named constant) for the switchover year rather than hard-coding 2022 in different places.

Reply:
After much deliberation, we went for `recruitment_cycle.after_2021?`